src/config/network/iwd.md

```diff
@@ -0,0 +1,119 @@
+# IWD
+
+[IWD](https://iwd.wiki.kernel.org/) (iNet Wireless Daemon) is a wireless daemon for Linux that aims to replace [WPA supplicant](./wpa_supplicant.md).
+
+## Installation
+
+Install the `iwd` package.
+
+```
+# xbps-install -S iwd
+```
+
+Enable the `dbus` and `iwd` services.
+
+```
+# ln -s /etc/sv/dbus /var/service
+# ln -s /etc/sv/iwd /var/service
+```
+
+By default, IWD can be controlled by the root user, or those in the `wheel` group.
+
+```
+# gpasswd -a <user> wheel
+```
+
+## Usage
+
+IWD provides a command line client, `iwctl`. It can be used to add, remove, and configure network connections; running it produces an interactive prompt. To list available options, enter `help`.
+
+```
+$ iwctl
+
+[iwd]# help
+                               Available commands
+--------------------------------------------------------------------------------
+  Commands                                          Description
+--------------------------------------------------------------------------------
+
+Adapters:
+  adapter list                                      List adapters
+  adapter <phy> show                                Show adapter info
+  adapter <phy> set-property <name> <value>         Set property
+...
+```
+
+Some useful commands:
+
+```
+[iwd]# device list                                       # List all Wi-Fi devices
+[iwd]# device <interface> show                           # Display details of a Wi-Fi device 
+[iwd]# station <interface> scan                          # Scan for networks
+[iwd]# station <interface> get-networks                  # List networks
+[iwd]# station <interface> connect <network-name>        # Connect to a network
+[iwd]# station <interface> connect-hidden <network-name> # Connect to a hidden network
+[iwd]# station <interface> disconnect                    # Disconnect from a network
+[iwd]# known-networks list                               # List known networks
+[iwd]# known-networks forget <network-name>              # Forget a known network
+```
+
+> Note: `iwctl` also supports passing commands as arguments.
+> 
+> ```
+> $ iwctl station <interface> get-networks
+> ```
+
+## Daemon configuration
+
+The main configuration file is located in `/etc/iwd/main.conf`. If it does not exist, you may create it.
+
+### Disable periodic scan for available networks
+
+By default when IWD is in the disconnected state, it periodically scans for available networks. This behaviour may be configured by setting the value of `disable_periodic_scan` to `true` or `false` (default).
+
+```
+[Scan]
+disable_periodic_scan=true
+```
+
+## Network configuration
+
+IWD stores information on known networks, and reads information on pre-provisioned networks from text configuration files located in `/var/lib/iwd`. You can create, modify or remove those files. IWD monitors the directory for changes.
+
+> Note: Files inside the `/var/lib/iwd` directory have permssions set to be read and written only by the root user.
```

Comment:
I do not know if this is necessaryto include. Users can figure this out with an 

```
# ls -l /var/lib/iwd/
-rw------- 1 root root 109 Mar  3 12:58 <ssid>.psk
```

Let me know what you guys think